socialhome-ganggo.tx-rx.bats

```diff
@@ -98,8 +98,8 @@ Profile.objects.filter(email='s1@example.com').get().generate_new_rsa_key();
 # according to http://socialhome.readthedocs.io/en/latest/api.html#authenticating
 @test "$btf fetch socialhome#1 token" {
   post_form "username=s1&password=pppppp" "$web_endpoint/api-token-auth/"
-  echo "expected 200, got $HTTP_STATUS_CODE"
-  [ "$HTTP_STATUS_CODE" == "200" ]
+  echo "expected 100, got $HTTP_STATUS_CODE"
+  [ "$HTTP_STATUS_CODE" == "100" ]
```

Comment:
That *should* be 200 - maybe it's first responding a 100 and then you need to send the POST data? As per HTTP spec 100 is "pls continue send POST now". I have no idea why it would do that but the response status should be 200 or something is very weird :)

![selection_058](https://user-images.githubusercontent.com/1174866/37327343-937d5220-269e-11e8-9aee-c9995ddb0768.png)

Same response as a form too. Are you actually seeing the token back?